charts/victoria-metrics-operator/CHANGELOG.md

```diff
@@ -2,7 +2,10 @@
 
 ## Next release
 
-- TODO
+- Renamed `cleanupCRD` to `cleanupCRD.enabled`
+- Renamed `cleanupImage` to `cleanupCRD.image`
+- Replaced `watchNamespace` string with `watchNamespaces` slice
+- Removed unnecessary Role and RoleBinding
```

Comment:
It'd be great to have a guide or example with minimal operator RBAC permissions for single/multi namespace deployment.

Reply:
Also, operator is always requires single namespace permissions for leader-election mechanism.

Reply:
reverted role, but left permissions for leader election only